src/commands/appSettings/downloadAppSettings.ts

```diff
@@ -13,67 +13,52 @@ import { ext } from "../../extensionVariables";
 import { getLocalSettingsJson, ILocalSettingsJson } from "../../funcConfig/local.settings";
 import { localize } from "../../localize";
 import * as api from '../../vscode-azurefunctions.api';
-import { confirmOverwriteSettings } from "./confirmOverwriteSettings";
 import { decryptLocalSettings } from "./decryptLocalSettings";
 import { encryptLocalSettings } from "./encryptLocalSettings";
+import { filterDownloadAppSettings } from "./filterDownloadAppSettings";
 import { getLocalSettingsFile } from "./getLocalSettingsFile";
 
-// Valen: Imported in registerCommands
 export async function downloadAppSettings(context: IActionContext, node?: AppSettingsTreeItem): Promise<void> {
-    // Valen: node is an optional parameter so this checks if it is null to set a value to it.
     if (!node) {
         node = await ext.tree.showTreeItemPicker<AppSettingsTreeItem>(AppSettingsTreeItem.contextValue, context);
     }
 
     const client: IAppSettingsClient = node.client;
```

Comment:
there is no need to reassign.. you can just inline node.client where it is being used below.